api/src/main/java/org/jboss/forge/roaster/Roaster.java

```diff
@@ -96,25 +103,54 @@
    /**
     * Open the given {@link File}, parsing its contents into a new {@link JavaType} instance.
     */
+   @Deprecated
```

Comment:
No need to deprecate these methods. The default charset works for the majority of use cases